src/lib/mpq/chain.js

```diff
@@ -27,7 +27,7 @@ class MPQChain {
       path.join(dataDir, mpq)
     ));
 
-    const archives = glob.sync(patterns);
+    const archives = glob.sync(patterns, {nocase: true});
```

Comment:
For which scenario is this a requirement? Are MPQs shipped with variants in casing?

Reply:
Make sure to run `npm run lint` to comply with the linting configuration for this project. In this case, we require spacing within object literals.

Reply:
Soo, for example from torrent sometime I get with random case :) Mostly it for custom clients, I help for some private server project, which has big count of client changes (races, quests, spells, models and etc) and you database API is helpful for it (but not working properly at this moment, custom races not properly show and quests eat CPU and memory but nothing shown) and yes, all mpq`s in lower case

Reply:
Sounds good to me. If you could add the spacing, that'd be great 👍 

As for custom races not properly showing and quests consuming CPU / memory, please open separate issues. There's likely things that can be done there.

Reply:
I will make changes for you review in 15-20 hours, have some other work.